src/wasm-lib/justfile

```diff
@@ -40,9 +40,9 @@ overwrite-sim-test test_name:
     EXPECTORATE=overwrite {{cita}} -p kcl-lib -- simulation_tests::{{test_name}}::parse
     EXPECTORATE=overwrite {{cita}} -p kcl-lib -- simulation_tests::{{test_name}}::unparse
     {{cita}} -p kcl-lib -- tests::{{test_name}}::kcl_test_execute
-  
+
 test:
-    export RUST_BRACKTRACE="full" && cargo nextest run --workspace --test-threads=1
+    export RUST_BRACKTRACE="full" && cargo nextest run --workspace --no-fail-fast --profile ci
```

Comment:
Why use the CI profile? I made the CI profile because CI is usually slower than our dedicated local machines, so the nextest timeout should be a bit more generous.

Reply:
ah okay fair but test-threads 1 is way too limiting like we know what can be run in parallel

Reply:
can i make a new profile?

Reply:
im happy now. haha